Xamarin.Forms.Platform.iOS/Xamarin.Forms.Platform.iOS.csproj

```diff
@@ -47,6 +47,9 @@
       <Version>60.1.0</Version>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup>
+    <BundleResource Include="Resources\3bar%402x.png" />
```

Comment:
BundleResource is the same as Embedded resource?

Reply:
No. BundleResource is Apple specific. This looks right.